flask_tutorial-code_review/migrations/alembic.ini

```diff
@@ -0,0 +1,50 @@
+# A generic, single database configuration.
```

Comment:
remove migrations folder and unnessecary files

Reply:
done